app/views/admin/borrowings/_form_search.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= search_form_for [:admin, @q], class: "form-inline" do |f| %>
+  <div class="form-group">
+    <span><%= t ".borrow_code" %></span>
+    <%= f.search_field :borrow_code_cont,
+      class: "form-control code_value",
+      placeholder: t(".borrow_code") %>
+  </div>
+  <div class="form-group">
+    <span><%= t ".user_name" %></span>
+    <%= f.search_field :user_name_cont,
+      class: "form-control code_value",
+      placeholder: t(".user_name") %>
+  </div>
+  <div class="form-group">
+    <span><%= t ".date_borrow" %></span>
+    <%= f.date_field :date_borrow_gteq, class: "form-control" %>
```

Comment:
gteq là gì

Reply:
gteq(greater than or equal to) : lớn hơn hoặc bằng, ngược lại với lteq chị ạ